src/platform/plugins/shared/data/common/serialize_utils.test.ts

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import { MultiFieldKey, RangeKey } from '.';
+import { SerializedRangeKey } from './search';
+import { SerializedMultiFieldKey } from './search/aggs/buckets/multi_field_key';
+import { SerializableType, deserializeField, serializeField } from './serialize_utils';
+
+describe('serializeField/deserializeField', () => {
```

Comment:
Should we also test some `unknown` values? like the array that I've mentioned in a comment above?

Reply:
Yes, I'm fine to do that but only if there is a way to have array of values outside of the **Last value** agg.